dayrepo/snippets/views.py

```diff
@@ -291,15 +291,14 @@ def post(self, request, snippet_id):
             trouble_cause=req.get("trouble_cause"),
             trouble_support=req.get("trouble_support"),
         )
-        post_trouble.delete()
         duties_trouble.save()
 
         # 工程テーブルフォーム保存
-        form_process_count = len(req.getlist("via_point"))
-        for i in range(form_process_count):
+        def_count = len(post_process)
+        for i in range(def_count):
             process = Process(
-                id=post_process[i].id,
-                snippet_id=post_process[i].snippet_id,
+                id= post_process[i].id,
```

Comment:
他のところにスペースないからここも消すかー
```suggestion
                id=post_process[i].id,
```

Reply:
対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/72/commits/a733989a012feba20f5fb267012cb6483913f226